src/Umbraco.Infrastructure/Services/Implement/FileService.cs

```diff
@@ -318,9 +320,9 @@ public long GetScriptFileSize(string filepath)
                 // This fixes: http://issues.umbraco.org/issue/U4-7953
                 contentTypeName);
 
-            var evtMsgs = EventMessagesFactory.Get();
+            EventMessages eventMessages = EventMessagesFactory.Get();
 
-            // TODO: This isn't pretty because we we're required to maintain backwards compatibility so we could not change
+            // TODO: This isn't pretty because we're required to maintain backwards compatibility so we could not change
             // the event args here. The other option is to create a different event with different event
             // args specifically for this method... which also isn't pretty. So fix this in v8!
```

Comment:
I guess we should fix this TODO now, instead of having the Additional Data?

Reply:
Or we could of course add a `CreateTemplateForContentType` and a `ContentTypeAlias` field to the Saving/Saved Notification, I'm not sure what's better?

Reply:
I think we both are better options then the current :) But maybe a hybrid, where we have properties that use the state as backing field, so we are sure the values are kept between the Saving...Saved notifications?

Reply:
Oh that's a great idea :D